cmd/members/invite.go

```diff
@@ -61,15 +62,25 @@ func runInvite(client members.Client) func(*cobra.Command, []string) error {
 			memberInputs,
 		)
 		if err != nil {
-			return err
+			output, err := output.CmdOutputResource(
+				viper.GetString(cliflags.OutputFlag),
+				[]byte(err.Error()),
+				output.ErrorPlaintextOutputFn,
+			)
+			if err != nil {
+				return errors.NewError(err.Error())
+			}
+
+			return errors.NewError(output)
 		}
 
-		output, err := output.CmdOutput(
+		output, err := output.CmdOutputResources(
 			viper.GetString(cliflags.OutputFlag),
-			output.NewSingularOutput(response),
+			response,
+			output.MultipleEmailPlaintextOutputFn,
```

Comment:
Here's an example of a different function used because the response is
```
[
    {
        "email": ""
    }
]
```
which is different from a normal "list" type of response because
1. it doesn't have "items" as a property
2. it has "email" and "_id" instead of "name" and "key"